examples/annotation_import/image.ipynb

```diff
@@ -1,18 +1,16 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #33.          "dataRow": {
</pre><p>This shouldn't be place here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>